pkg/catalog/retry_test.go

```diff
@@ -14,6 +15,12 @@ import (
 	"github.com/openservicemesh/osm/pkg/k8s"
 	"github.com/openservicemesh/osm/pkg/policy"
 	"github.com/openservicemesh/osm/pkg/service"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	thresholdUintVal  uint32 = 3
+	thresholdDuration        = metav1.Duration{Duration: time.Duration(1 * time.Second)}
 )
```

Comment:
nit: For consistency with other tests, these variables could be declared in the `TestGetRetryPolicy` method